apisix/core/config_etcd.lua

```diff
@@ -314,36 +316,44 @@ local function sync_data(self)
             res.value.id = key
             self.values[pre_index] = res
             res.clean_handlers = {}
+            log.info("update data by key: ", key)
 
         else
             self.sync_times = self.sync_times + 1
             self.values[pre_index] = false
+            self.values_hash[key] = nil
+            log.info("delete data by key: ", key)
         end
 
     elseif res.value then
         res.clean_handlers = {}
         insert_tab(self.values, res)
         self.values_hash[key] = #self.values
         res.value.id = key
+        log.info("insert data by key: ", key)
     end
 
     -- avoid space waste
     -- todo: need to cover this path, it is important.
```

Comment:
Is it still a TODO?

Reply:
we can remove this comment now. we have added test cases about it.